_includes/search/search_form.html

```diff
@@ -2,11 +2,19 @@
 {%- assign search_provider = site.search_provider | default: "lunr" -%}
 {%- case search_provider -%}
   {%- when "lunr" -%}
-    <input type="search" id="search" aria-label="{{ site.data.ui-text[site.locale].search_label_text | default: 'Enter your search term...' }}" class="search-input" tabindex="-1" placeholder="{{ site.data.ui-text[site.locale].search_placeholder_text | default: 'Enter your search term...' }}" />
+    <form>
```

Comment:
If you wrap the input with a `<form></form>` element, hitting <kbd>Enter</kbd> tries to submit the form, which reloads the page. This is why it's not included currently since it's not actually submitting anything.

Though I'm not sure if that's correct semantically or not. Can you have an `input` and `label` without inclosing them in a `form`?

If you do, then we might need to add some JavaScript to disable the default behavior that reloads the page when hitting <kbd>Enter</kbd>.

Reply:
Looks like it's cool to have them outside of a `form` element.

https://stackoverflow.com/questions/8568660/is-there-any-danger-to-using-input-fields-outside-without-forms-in-html-javascri

Reply:
Resolved this now, it's inside a form, i've gone for a slight ghetto onkeydown solution to prevent keypress, didnt seem to warrant an external file.